### PR TITLE
Fixes bad global used by turf emote

### PR DESCRIPTION
**Pull request by Tastyfish**
## About The Pull Request

The turf emote was using an unmanaged global to track which turf was picked, making it entirely luck that a given user spawned the correct object if 2 people were using *turf at once.

## How This Contributes To The Skyrat Roleplay Experience

Unmanaged globals violate code standards anyway, but this shouldn't be global in any form.

## Changelog

:cl:
fix: Fixed possible edge case issues with multiple uses of turf emote at the same time.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/emotes/code/additionalemotes/turf_emote.dm [hunk 1]
```diff
@@ -8,6 +6,7 @@ var/current_turf
 	key = "turf"
 	key_third_person = "turf"
 	cooldown = 4 SECONDS
+	var/current_turf
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/emotes/code/additionalemotes/turf_emote.dm:10]: Just document this please!

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/emotes/code/additionalemotes/turf_emote.dm (hunk 1 above)
Comment (Tastyfish) [reply modular_skyrat/modules/emotes/code/additionalemotes/turf_emote.dm:10]: yep, done

File: modular_skyrat/modules/emotes/code/additionalemotes/turf_emote.dm [hunk 2]
```diff
@@ -94,8 +94,7 @@ var/current_turf
 	if(do_after(user,10))
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/emotes/code/additionalemotes/turf_emote.dm]:
```suggestion
	if(do_after(user, 1 SECONDS))
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Once that last little comment is done, we're good to go.

**Pull request merged**